src/components/molecules/pager/pager.scss

```diff
@@ -1,15 +1,14 @@
-@import '../../../styles/breakpoints';
-
-.pager__navigation,
 .pager__container {
-	align-items: center;
-	display: flex;
-	justify-content: flex-start;
 	margin-left: 0;
-}
+	font-family: Noto Sans, Century Gothic, Arial, sans-serif;
```

Comment:
it shouldn't go here either.   The font should be part of theming and the pager should use the application's base font unless overridden in the view